decode.go

```diff
@@ -396,3 +398,18 @@ func floatSetter(bitSize int) valueSetter {
 		return nil
 	}
 }
+
+func boolSetter(v reflect.Value, raw rawValue) error {
+	if len(raw.data) == 0 {
+		return nil
+	}
+
+	trimmedBool := strings.TrimSpace(raw.data)
```

Comment:
Is this trimming necessary? I believe the raw value is already trimmed by the time it reaches the setter functions.

Reply:
It’s not, my bad. You are right, removed it. It’s trimming before it comes here